Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM python:3.6-alpine
 
 COPY requirements.txt app/
 
+RUN apk --no-cache add gcc musl-dev
```

Comment:
Why do we need a compiler? If it is required by a Python dependency?
If so, the fancy thing to do here is to install the compiler, install the dependency and finally uninstall the compiler as a last step (all within one RUN statement).